gradle.properties

```diff
@@ -57,3 +57,4 @@ slf4jVersion=2.0.0-alpha7
 storageBlobVersion=12.19.1
 swagger=2.2.2
 websocketVersion=2.0.0
+org.gradle.jvmargs=-Xmx4608m
```

Comment:
I would remove this, since other people could not have different settings. I had the same problem
you could place this config in your `~/.gradle/gradle.properties` and it should work

Reply:
Good advice, i agree with you.